### DISCUSSION TITLE
conflict between react-flow and react-color

### DISCUSSION THREAD

**Discussion by amintabrizi** [xyflow/xyflow#2711] [General]
I can not use react-flow and react-color together. in fact, This error occurs when I want to drag between colors and pick a new color. Can somebody help me?

**Comment by bcakmakoglu** [comment]
A sandbox or any other reproduction would be helpful otherwise it's sort of hard to help you 😄

**Comment by amintabrizi** [comment]
as you can see, we can not select new color from color picker box!
https://codesandbox.io/s/keen-snow-5prhsd

**Comment by bcakmakoglu** [comment]
Well, there's no "conflict" at all.
First, you need to add `nodrag` to either the color picker itself, or wrap it with a div or something and add that class.
Second, you need to pass in a color to the picker and handle the change event.

[Here's a corrected sandbox.](https://codesandbox.io/s/bold-hamilton-91kq7w?file=/src/App.js)

**Comment by amintabrizi** [comment]
Thanks a lot my friend.
I spent a few hours for the problem and finally find the problem.
if I use [NodeToolbar ](https://reactflow.dev/docs/api/nodes/node-toolbar/) in my project, react-color wont work.please see in below link:
https://codesandbox.io/s/cool-swartz-eb1gzs?file=/src/CustomNode.js
as you can see the top color picker works because it is not inside of NodeToolbar and also the bottom color picker does not work because it is inside NodeToolbar.
For seeing the bottom color picker please click in a button.
I hope to you can help me :)

**Comment by amintabrizi** [comment]
@moklick @bcakmakoglu
Could you help me?

**Comment by amintabrizi** [comment]
@moklick @bcakmakoglu 
Can you answer me?
I want to choose another solution if it is not possible to fix the problem

**Reply by bcakmakoglu** [reply]
Hey sorry, I'm a bit short on time as I actually have a job to do as well 😅
I can look into it at some point but I can't help you on-call.

**Reply by amintabrizi** [reply]
So I have to find another solution for this problem, Thanks any way :)

**Reply by rm-novaro** [reply]
I had the same issue. "nodrag" class is not helping when the react-color is in NodeToolbar.
@amintabrizi Did you find any solution ?
